### PR TITLE
fixing grammar future tense to present tense, removing pronouns

### PR DESCRIPTION
**Pull request by rlopez133**
Fixing some grammatical mistakes, removing use of future tense wording and keeping it present tense i.e. will. Removed use of pronouns i.e. you, your. Included sudo to the diagnostic commands as a certain amount of privilege is required to run the commands.

### CODE REVIEW

**Review by kalexand-rh — COMMENTED** [review]
I have more changes. :)

**Comment by vikram-redhat** [comment]
@rlopez133 - will you be updating this PR based on @kalexand-rh's changes? Please make sure to rebase before you do.

File: admin_guide/diagnostics_tool.adoc [hunk 1]
```diff
@@ -31,33 +31,46 @@ connected to.
 [[admin-guide-using-the-diagnostics-tool]]
 == Using the Diagnostics Tool
 
-{product-title} can be deployed in many ways: built from source, included in a
-VM image, in a container image, or as enterprise RPMs. Each method implies a
-different configuration and environment. To minimize environment assumptions,
-the diagnostics were added to the `openshift` binary so that wherever there is
-an {product-title} server or client, the diagnostics can run in the exact same
-environment.
+{product-title} may be deployed in numerous scenarios including:
```
[the remark this answers is not in the document]
Comment (rlopez133) [reply admin_guide/diagnostics_tool.adoc]: Do we want to start sentences with You?

**Comment by bfallonf** [comment]
Sorry, I should've noted that this PR came before my PR of #8741  and this was left open so we could discuss it (over email), and I created #8741 out of that discussion. I'd suggest this PR be closed.

That said, I can see that there was a rebase (that probably took time) so it's up to @rlopez133 and maybe @vikram-redhat to determine what to do.

**Comment by vikram-redhat** [comment]
@bfallonf - is this undoing any of the work in 8741?

**Comment by bfallonf** [comment]
@vikram-redhat Merging this would undo some of 8741, yes.

**Comment by rlopez133** [comment]
@vikram-redhat @bfallonf - Also when I did the rebase, I did do a "this sounds the best" when having to make some decisions... I won't be offended if you want to change it back or to something else. Probably my biggest thing is starting sentences with pronouns like "You" but to each his own :)

**Comment by bfallonf** [comment]
I'll paste the above msg from @kalexand-rh here, cos it's worded well:

> Yes. Our style guide says that docs are supposed to favor active, user-focused sentences. It's always better to say "you," meaning the user, "do <the thing>" instead of "<The thing> can be done."

Big thanks again @rlopez133 ! I'll do a quick followup PR.

**Comment by bfallonf** [comment]
Followup PR at #10983 

I'll close this PR. If anyone has thoughts, please let me know.

**Pull request closed without merging**